routes/api/accounts.js

```diff
@@ -0,0 +1,53 @@
+const express = require( 'express' )
+
+const bcrypt = require( 'bcrypt' )
+const encryptPassword = require( '../../auth/encryptPassword' )
+const RegistrationEmail = require( '../../src/mail/registration_email.js' )
+const jwt = require( 'jsonwebtoken' )
+const { testForCode, whereClause } = require( '../accounts/verify_user' )
+
+const router = express.Router()
+
+const ERROR_MESSAGE = "Incorrect email or password. Have you verified your email?"
+const ERROR_MESSAGEB = "Email not valid"
+
+router.post( '/register', ( request, response, next ) => {
+  const { User } = request.app.get( 'models' )
+  const { email, password } = request.body
+
+  User.create({ email, password: encryptPassword( password ) })
+    .then( user => RegistrationEmail.send( user ) )
+    .then( user => response.status( 202 ).json( { message: 'K thanx go check ur email' } ) )
+    .catch( err => response.status( 401 ).json( { err, success: false, message: ERROR_MESSAGEB  } ) )
+})
+
+router.post('/authenticate', ( request, response ) => {
+  const { email, password } = request.body
+
+  const { User } = request.app.get('models')
+
+  User.findOne({ where: { email, email_verified: true }})
+    .then( user => {
+
+      bcrypt.compare( password, user.password, (error, result ) => {
+
+        if( result ) {
+          const token = jwt.sign( { user }, 'superSecret', {
+            expiresIn: "7 days" // expires in 24 hours
+          })
+
+          response.json({
+            success: true,
+            token: token,
```

Comment:
Use implicit object creation (I also renamed the `user` key to be `user_id` since that is a little more accurate in this context): 
```js
response.json({ success: true, token, user_id: user.id, email })
```